benchmarks/benchmark/tools/locust-load-inference/variables.tf

```diff
@@ -71,6 +71,12 @@ variable "num_locust_workers" {
   default     = 1
 }
 
+variable "stop_timeout" {
+  description = "Length of time before a locust job is stopped."
+  type        = number
+  default     = 0
```

Comment:
Default should be 10800 to keep consistent behavior. Or is 0 a more reasonable default in this case?

Reply:
I chose `0`, as that forces this flag to be opt-in. My thinking is making the field opt-in reduces the chance of hitting: https://github.com/locustio/locust/issues/2621. Which I hit during development

LMKWYT

Reply:
I tested locally with 0 with my current tests and confirmed it has no negative impact. Also looks like stop_timeout doesn't do anything in terms of stopping long running user tasks as I had thought - it actually impacts "graceful termination" - https://github.com/locustio/locust/blob/a2f328fdb2fcf5592b07b737680a9edcfa201301/locust/env.py#L83

Reply:
With that - I'm fine with this default set to 0 :) thanks for explaining and fixing this!